design/user_types.go

```diff
@@ -129,3 +129,17 @@ var UpdateTrackerQueryAlternatePayload = a.Type("UpdateTrackerQueryAlternatePayl
 	})
 	a.Required("query", "schedule", "trackerID")
 })
+
+// IdentityDataAttributes represents an identified user object attributes
+var IdentityDataAttributes = a.Type("IdentityDataAttributes", func() {
+	a.Attribute("fullName", d.String, "The users full name")
+	a.Attribute("imageURL", d.String, "The avatar image for the user")
+})
+
+// IdentityData represents an identified user object
+var IdentityData = a.Type("IdentityData", func() {
```

Comment:
I would just call this type "Identity". It's the resources object for an identity. That doesn't depend on the context, really.